python-avd/pyavd/_eos_designs/structured_config/core_interfaces_and_l3_edge/utils.py

```diff
@@ -211,6 +211,28 @@ def _get_common_interface_cfg(self: AvdStructuredConfigCoreInterfacesAndL3Edge,
         if p2p_link.ip:
             interface_cfg["ip_address"] = p2p_link.ip[index]
 
+        if get(p2p_link, "ptp.enabled"):
+            ptp_config = {}
+
+            if self.shared_utils.ptp_enabled:
+                # Apply PTP profile config from node settings when profile is not defined on p2p_link
+                if (ptp_profile_name := get(p2p_link, "ptp.profile")) is None:
```

Comment:
```suggestion
                if not p2p_link.ptp.profile:
```

Reply:
This doesnt work when `ptp_profiles` are defined and any interface is applied with default ptp profile `aes67-r16-2016`. It tries to find this default profile in available ptp profiles.